DOCS/xml-fields.md

```diff
@@ -0,0 +1,177 @@
+* `isOsiApproved` (for licenses, not exceptions): either "true" or "false" based on whether this license has been approved by the [Open Source Initiative](https://opensource.org/licenses/alphabetical)
+* `listVersionAdded`: in which release version of the SPDX License List was the license first added, e.g. "3.19"
+
+Finally, if the license ID has been deprecated, two additional attributes should be included:
+* `isDeprecated`: should be "true" if the license ID is deprecated
+* `deprecatedVersion`: in which release version of the SPDX License List was the license first marked as deprecated, e.g. "3.19"
+
+Note that the deprecated tags refer to whether the ID has been deprecated _by SPDX_ -- in other words, if the SPDX Legal Team no longer recommends that the identifier should be used. Licenses that have been described as "deprecated" or "superseded" by the _license author_ might no longer be recommended to use for new code, but their identifiers remain valid License IDs on the SPDX License List.
+
+### License metadata
+
+There are three optional first-level metadata fields that can be nested immediately under `<license>` or `<exception>`, each of which is optional:
+
+* **`<crossRefs>`**: Defines cross-references to URLs where the license can be found in the wild
+* **`<notes>`**: Describes general comments about the license
+* **`<obsoletedBys>`** (if license is deprecated): Lists the license(s) that should be used instead of this deprecated license
+
+### License text
+
+Then, the actual license text would be contained within the `<text></text>` section:
+
+* **`<text>`**: Defines the actual text and templating for the license
+
+Example:
+
+```xml
+<license licenseId="MyLicense" name="My License">
+    <crossRefs>
+        <crossRef>https://example.com/mylicense.md</crossRef>
+	<crossRef>https://example.com/MyCode.c</crossRef>
+    </crossRefs>
+    <notes>This license is a modified version of MyOtherLicense, with an extra paragraph at the end.</notes>
+    <text>
+        [Actual text goes here...]
+    </text>
+</license>
+```
+
+## Template fields
+
+### Section annotation markers
+
+Some portions of the license text can be surrounded by annotation tags, to denote certain sections of the license text:
+
+* **`<titleText>`**: indicates the title of the license, if specified in the license text itself
+* **`<copyrightText>`**: indicates where a copyright notice would be placed for the licensed code
+  * Note that `<copyrightText>` should _not_ be used for copyright notices that apply to the copyright in the license text itself!
+* **`<standardLicenseHeader>`**: indicates a standard way that the license recommends specifying the license, e.g. in code comments
+  * Note that `<standardLicenseHeader>` should _only_ be used if it is defined by the license text itself!
+  * Typically, this would be located in an appendix titled something like "How to apply the license to your work"
```

Comment:
Maybe add 'Do not use it if the license text is included verbatim in the file' perhaps?

Reply:
I'm inclined to say that if we want to add more clarity about what SPDX considers to be a "standard license header", that would probably be more appropriate to add to other documentation / FAQ and then point to it from here. I'm hesitant to expand the scope of this document too much (I'm pretty wordy already as it is...)